Add embedded YouTube video to histogram docs (#1186)
timescale/docs: timescaledb/tutorials/grafana/visualizations/histograms.md

  Grafana variables. You change the values of these variables by
  using the dashboard's date chooser when viewing your graph.

> Check out the video below for a step-by-step walk through on how to create 
> histograms in Grafana.
> <video url="https://www.youtube-nocookie.com/embed/h1eTIYOFplA"/>

Reviewer: ```suggestion
Check out this video for a step-by-step walkthrough on creating
histograms in Grafana:
<video url="https://www.youtube-nocookie.com/embed/h1eTIYOFplA"/>
```

will show you the message: 
`Results have been limited to 1000000 because the SQL row limit was reached`

> This means Grafana has decided not to display all rows returned by the query. To

Reviewer: ```suggestion
This means Grafana is not displaying all rows returned by the query. To
```

> solve this problem, pre-aggregate the data in your query using TimescaleDB's 

Reviewer: ```suggestion
solve this problem, pre-aggregate the data in your query using the TimescaleDB
```

    <img class="main-content__illustration" src="https://assets.timescale.com/docs/images/tutorials/visualizations/histograms/volume_distribution.png" alt="Screenshot of Grafana histogram showing the stock volume distribution for $AMZN."/>
    
    The plot shows a left-skewed distribution for the AMZN symbol. For many
    symbols, you might see a distorted distribution, due to outliers 
    representing a few very large volume transactions. There are 2 solutions:

>     1.  Limit your query to transactions with volumes less than a certain 
>         threshold.
>     1.  Use a logarithmic scale. Unfortunately these are not yet supported in 

Reviewer: ```suggestion
    *  Limit your query to transactions with volumes less than a certain 
        threshold.
    *  Use a logarithmic scale. Unfortunately these are not yet supported in 
```

The test is to only use numbers when order is important.
